apps/server/src/shared/domain/entity/external-tools/external-tool/config/oauth2-tool-config.ts

```diff
@@ -7,21 +7,13 @@ export class Oauth2ToolConfig extends ExternalToolConfig {
 	@Property()
 	clientId: string;
 
-	@Property()
-	clientSecret: string;
```

Comment:
We dont save it, why we encrypted it in the uc? shouldnt hydra do that for us?

Reply:
done